src/main/java/Duke.java

```diff
@@ -1,10 +1,50 @@
+import java.util.Scanner;
 public class Duke {
+    static TaskManager tasks = new TaskManager();
+    static String PRINT_LINE = "____________________________________________________________\n";
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+       /*
+       String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        */
+
+        System.out.println(
+                PRINT_LINE +
+                        " Hello! I'm Duke\n" +
+                        " What can I do for you?\n" +
+                        PRINT_LINE
+        );
+        Scanner sc = new Scanner(System.in);
+        boolean isFinished = false;
+        int taskNum = 0;
+
+
+        do {
+            String command = sc.nextLine();
+
+            if(command.equals("bye")){
+                isFinished = true;
+            } else if(command.startsWith("mark")){
+                taskNum = Integer.parseInt(command.substring(command.length()-1));
+                tasks.markTask(taskNum);
+            } else if(command.startsWith("unmark")){
+                taskNum = Integer.parseInt(command.substring(command.length()-1));
+                tasks.unmarkTask(taskNum);
+            } else if(command.equals("list")){ // list
```

Comment:
This if-else violates the code standards in terms of spacing. It should be: `} else if (command.equals("list")) {`.